website_sale_with_tax/views/product_item_view.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+<data>
+
+    <template id="products_item_with_tax" inherit_id="website_sale.products_item">
+
+        <xpath expr="//div[@class='product_price']/b/span[1]" position="attributes">
```

Comment:
Better use `<span itemprop="price"...` for finding the element (mainteinability would be better)